.rubocop_todo.yml

```diff
@@ -16,7 +16,7 @@ Metrics/AbcSize:
 # Offense count: 2
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 208
+  Max: 214
```

Comment:
This was done before I split things out into modules, it can be removed
now. ;-)

On Friday, July 17, 2015, Brian Underwood <notifications@github.com> wrote:

> In .rubocop_todo.yml
> <https://github.com/neo4jrb/neo4j/pull/871#discussion_r34912584>:
>
> > @@ -16,7 +16,7 @@ Metrics/AbcSize:
> >  # Offense count: 2
> >  # Configuration parameters: CountComments.
> >  Metrics/ClassLength:
> > -  Max: 208
> > +  Max: 214
>
> Tsk tsk... ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/neo4jrb/neo4j/pull/871/files#r34912584>.
>